src/main/java/animals/StorageService.java

```diff
@@ -0,0 +1,64 @@
+package animals;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public enum StorageService {
```

Comment:
For me, that's weird that the enum contains `Service` in the name. Can you think about a better name?